cargo/Cargo.toml

```diff
@@ -62,6 +62,18 @@ additional_flags = ["--cfg=atomic_cas"]
 # automatically add non-`*.rs` files to the Bazel workspace.
 # https://github.com/google/cargo-raze/issues/41
 [raze.crates.ring.'0.16.12']
+gen_buildrs = true
+additional_flags = [
+  # These flags are necessary to link a static `ring-core` library compiled from C sources.
+  #
+  # Do not change `remote/ring-0.16.12.BUILD` since it contains manually added evnironment variables
+  # that were added to solve #890:
+  # https://github.com/project-oak/oak/issues/890#issuecomment-621773356
+  # `cargo raze` can change `remote/ring-0.16.12.BUILD`, so before compiling run:
+  # git checkout cargo/remote/ring-0.16.12.BUILD
```

Comment:
Maybe worth adding a change (or a big warning) to `scripts/cargo_raze` ?

(We should also change the comments in `{oak_abi,oak_runtime}/Cargo.toml` to mention using the script rather than raw `cargo raze`)

Reply:
Done